rhel8/profiles/ospp.profile

```diff
@@ -5,1220 +5,540 @@ title: 'Protection Profile for General Purpose Operating Systems'
+    - ensure_redhat_gpgkey_installed
+    - ensure_gpgcheck_globally_activated
+    - ensure_gpgcheck_local_packages
+    - ensure_gpgcheck_never_disabled
 
-    ## Add nosuid Option to /tmp
-    - mount_option_tmp_nosuid
+    ### Passwords
+    - var_password_pam_difok=4
+    - accounts_password_pam_difok
+    - var_password_pam_maxrepeat=3
+    - accounts_password_pam_maxrepeat
+    - var_password_pam_maxclassrepeat=4
+    - accounts_password_pam_maxclassrepeat
 
-    ###########
-    ## /swap
-    ###########
-    ## TO DO: https://github.com/ComplianceAsCode/content/issues/4490
-    ##  do we need a swap partition (for security reasons)?
-    #logvol swap --name=lv_swap --vgname=VolGroup --size=2016
+    ### Kernel Config
+    ## Boot prompt
+    #- package_dracut-fips_installed
+    - grub2_audit_argument
+    - grub2_audit_backlog_limit_argument
+    - grub2_slub_debug_argument
+    - grub2_page_poison_argument
+    - grub2_vsyscall_argument
+    - grub2_pti_argument
 
-    ###########
-    ## /dev/shm
-    ###########
-    ## Make sure /dev/shm is restricted
-    #echo -e "tmpfs\t/dev/shm\t\t\t\ttmpfs\tdefaults,mode=1777,,,strictatime\t0 0" >> /etc/fstab
+    ## Security Settings
+    - sysctl_kernel_kptr_restrict
+    - sysctl_kernel_dmesg_restrict
+    - sysctl_kernel_kexec_load_disabled
+    - sysctl_kernel_yama_ptrace_scope
+    - sysctl_kernel_perf_event_paranoid
+    - sysctl_user_max_user_namespaces
+    - sysctl_kernel_unprivileged_bpf_disabled
+    - sysctl_net_core_bpf_jit_harden
 
-    ## Add nodev Option to /dev/shm
-    - mount_option_dev_shm_nodev
+    ## File System Settings
+    - sysctl_fs_protected_hardlinks
+    - sysctl_fs_protected_symlinks
 
-    ## Add noexec Option to /dev/shm
-    - mount_option_dev_shm_noexec
+    ### Audit
+    - service_auditd_enabled
+    - var_auditd_flush=incremental_async
+    - auditd_data_retention_flush
+    - auditd_local_events
+    - auditd_write_logs
+    - auditd_log_format
+    - auditd_freq
+    - auditd_name_format
 
-    ## Add nosuid Option to /dev/shm
-    - mount_option_dev_shm_nosuid
+    ### Misc Audit Configuration
+    ### (not required in OSPP)
+    - audit_rules_session_events
+    - audit_rules_mac_modification
 
-    ###########
-    ## Misc Partitioning
-    ###########
-    ## Add nodev Option to Non-Root Local Partitions
-    - mount_option_nodev_nonroot_local_partitions
+    ### Module Blacklist
+    - kernel_module_usb-storage_disabled
+    - kernel_module_cramfs_disabled
+    - kernel_module_bluetooth_disabled
+    #- kernel_module_dccp_disabled
+    #- kernel_module_sctp_disabled
+    - kernel_module_firewire-core_disabled
+    - kernel_module_atm_disabled
+    - kernel_module_can_disabled
+    - kernel_module_tipc_disabled
 
-    #################################################################
-    ## Required Packages
-    #################################################################
+    ### rpcbind
+    - service_rpcbind_disabled
```

Comment:
Should be commented out (has not been present).

Reply:
This is on rhel7 config only.

Reply:
Updated.